pkg/csi_driver/multishare_controller.go

```diff
@@ -440,14 +470,19 @@ func (m *MultishareController) generateNewMultishareInstance(instanceName string
 		KmsKeyName:  kmsKeyName,
 		Labels:      labels,
 		Description: generateInstanceDescFromEcfsDesc(m.ecfsDescription),
-	}, nil
+	}
+	if m.featureMaxSharePerInstance {
```

Comment:
would it be better to condition on maxShareCount?
if maxShareCount == 0 ....

that way there's a easy way to specify "just use the default value"?

Reply:
I want to avoid indirect inference of the feature.